webapp/src/components/backstage/playbook_edit.tsx

```diff
@@ -386,30 +389,37 @@ const PlaybookEdit: FC<Props> = (props: Props) => {
             </BackstageNavbar>
             <Container>
                 <EditView>
-                    <EditHeader>
-                        <StagesAndStepsIcon/>
-                        <EditHeaderTextContainer>
-                            <EditHeaderText>{'Stages and Tasks'}</EditHeaderText>
-                            <EditHeaderHelpText>{'Stages allow you to group your tasks. Tasks are meant to be completed by members of the incident channel.'}</EditHeaderHelpText>
-                        </EditHeaderTextContainer>
-                    </EditHeader>
+                    <TabsHeader>
+                        <Tabs
+                            currentTab={currentTab}
+                            setCurrentTab={setCurrentTab}
+                        >
+                            {'Tasks'}
+                            {'Perferences'}
+                        </Tabs>
+                    </TabsHeader>
                     <EditContent>
-                        <StagesAndStepsEdit
-                            checklists={playbook.checklists}
-                            onChange={updateChecklist}
-                        />
+                        <TabsContent
+                            currentTab={currentTab}
+                        >
+                            <StagesAndStepsEdit
+                                checklists={playbook.checklists}
+                                onChange={updateChecklist}
+                            />
+                            {'TODO MM-30519 implement perferences.'}
```

Comment:
This is gone soon, but:
```suggestion
                            {'TODO MM-30519 implement preferences.'}
```